shimmingtoolbox/optimizer/lsq_optimizer.py

```diff
@@ -190,6 +215,14 @@ def _scipy_minimize(self, currents_0, unshimmed_vec, coil_mat, scipy_constraints
                                        jac=self._jacobian_func,
                                        options={'maxiter': 1000})
 
+        elif self.opt_criteria == 'ps_huber':
+            currents_sp = opt.minimize(self._criteria_func, currents_0,
+                                       args=(unshimmed_vec, coil_mat, factor),
+                                       method='SLSQP',
+                                       bounds=self.merged_bounds,
+                                       constraints=tuple(scipy_constraints),
+                                       options={'maxiter': 1000})
```

Comment:
Why do you need this? Isn't this the same code that would be run is the "else" statement?